css-styleguide.md

```diff
@@ -0,0 +1,652 @@
+directly in HTML classes.
+
+
+## Format
+
+### Spacing
+- Where possible, limit CSS files’ width to 80 characters.
+	- There will be unavoidable exceptions to this rule—such as URLs, or gradient syntax—which shouldn’t be worried about.
+- Use soft-tabs with a two space indent.
+- Put one space after : in property declarations.
+- Put spaces before { in rule declarations.
+- A blank line should be placed between each selector block.
+- Should end with a closing curly brace that is unindented and on a separate line
+- Each declaration should appear on its own line for more accurate error reporting.
+- Selectors should never be indented
+```scss
+// Good
+.rule {
+  margin: 3px;
+  text-align: center;
+}
+
+.another_rule {
+  margin: 3px;
+}
+
+// Bad
+.rule{
+    margin:3px;text-align:center;}
+```
+- Multiple selectors should each be on a single line, with no space after each
+comma.
+
+```scss
+selector1,
+selector2,
+selector3 {
+}
+```
+
+### Property-Value Pairs
+- Each on its own line
+- Be indented one level.
+- A single space after the colon that separates the name from the value.
+- End in a semicolon.
+```scss
+selector {
+  name: value;
+  name: value;
+}
+```
+
+- Strive to limit use of shorthand declarations to instances where you must
+explicitly set all the available values.
+```scss
+// Bad
+margin: inherit 3em;
+// Good
+margin-bottom: 3em;
+margin-top: 3em;
+```
+
+- URLs and string values should be single quoted
+```scss
+  background-image: url('/images/kittens.jpg');
+  font-family: 'Helvetica', sans-serif;
+  font-family: 'Lucida Grande', 'Helvetica', sans-serif;
+```
+- Top-level numeric calculations should always be wrapped in parentheses
+```scss
+// Good
+.component {
+  width: (100% / 3);
+}
+
+// Bad
+.component {
+  width: 100% / 3;
+}
+```
+- Avoid magic numbers. At the very least, comment them with a TODO
+```scss
+// Bad
+.component {
+  top: 0.327em;
+}
+
+// Better
+/**
+ * 1. Magic number. This value is the lowest I could find to align the top of
+ * `.foo` with its parent. Ideally, we should fix it properly.
+ */
+.component {
+  top: 0.327em;
+}
+
+// Good
+.component {
+  top: $align_top;
+}
+```
+
+### Sorting
+* Follow ordering:
+	1. variables
+	2. @extend directives
+	3. @include directives
+	4. properties
+- For properties alphabetical order or type order, just keep the whole project consistant.
+- Nested selectors always coming after a new line.
+- Mixin calls with @content coming after any nested selector.
+```scss
+// Good
+.module {
+  $amount = 3;
+  @extend .component;
+  @include sizing($amount);
+  margin-top: $amount * 1em;
+  text-align: center;
+
+  .module__ele {
+    color: #fff932;
+  }
+
+  @include media($sm) {
+    margin-top: $amount +10em;
+  }
+}
+```
+
+
+## Units
+### Measurements
+- Use **rem** units for font sizes with a em fallback.
```

Comment:
I've done this on Peace Corps and it's worked well so far, but I'd like more research done before it's implemented as a guide.